release/pkg/bundles/bundles.go

```diff
@@ -175,6 +175,14 @@ func GetVersionsBundles(r *releasetypes.ReleaseConfig, imageDigests map[string]s
 			return nil, errors.Wrapf(err, "Error getting bundle for vSphere infrastructure provider")
 		}
 
+		var nutanixBundle anywherev1alpha1.NutanixBundle
```

Comment:
this check can be moved out of this `for` loop since Nutanix bundle does not have separate artifacts for each Kubernetes version. There is a similar check for Snow bundle above on line 140, you can add it to that block.